packages/mobile/src/web3/saga.ts

```diff
@@ -129,8 +134,9 @@ export function* waitForWeb3Sync() {
 }
 
 export function* waitWeb3LastBlock() {
+  yield call(waitForGethConnectivity)
```

Comment:
If in forno mode will this hang indefinitely?

Reply:
no, the connection is set once geth starts up
ideally we would monitor to ensure the IPC remains open